### DISCUSSION TITLE
HeaderNavigation Multi level navigation

### DISCUSSION THREAD

**Discussion by reinisg** [carbon-design-system/carbon#9411] [Help] [answered]
Hello, 
I have few questions about UI-shell Header component.
1. Is it possible to use multi menu in HeaderNavigation, code example is below. There is a Sub Link 4 and it has childrens, now if I click menu closes and not opens child menu. Is there any property or there is no chance to do it? 
` <HeaderNavigation aria-label="IBM [Platform]">
              <HeaderMenuItem href="#">Link 1</HeaderMenuItem>

              <HeaderMenuItem href="#">Link 2</HeaderMenuItem>

              <HeaderMenuItem href="#">Link 3</HeaderMenuItem>

              <HeaderMenu aria-label="Link 4" menuLinkName="Link 4">
                <HeaderMenuItem href="#one">Sub-link 1</HeaderMenuItem>

                <HeaderMenuItem href="#two">Sub-link 2</HeaderMenuItem>

                <HeaderMenuItem href="#three">Sub-link 3</HeaderMenuItem>
                <HeaderMenu aria-label="Link 4" menuLinkName="Sub-Link 4">
                  <HeaderMenuItem href="#">Sub-Sub-link 1</HeaderMenuItem>

                  <HeaderMenuItem href="#">Sub-Sub-link 2</HeaderMenuItem>

                  <HeaderMenuItem href="#">Sub-Sub-link 3</HeaderMenuItem>
                </HeaderMenu>
              </HeaderMenu>
            </HeaderNavigation>`

2. And also is there possible to set HeaderMenu and SideNavMenu link? Or is it works only as submenu opener?

Regards Reinis

**Comment by jnm2377** [comment, marked as answer]
Hey @reinisg, unfortunately we don't support nested menus at this time. 
As for your second question, I'm not entirely sure I follow what you're asking. Could you show some examples or explain more?

**Reply by reinisg** [reply]
Thanks for the answer.
About second question, I mean if there any chance to set a link for HeaderMenu or SideNavMenu component.
Code example (something like this): HeaderMenu can be a property href ?
```
    <HeaderMenu aria-label="Link 4" menuLinkName="Link 4" href="#SOME_LINK_HERE">
         <HeaderMenuItem href="#one">Sub-link 1</HeaderMenuItem>
    </HeaderMenu>
```

**Reply by jnm2377** [reply]
Sorry for the late response. I understand now. Unfortunately, those components don't function as links, just as menu togglers. If you think this could be a useful feature or have seen this as a UX pattern in other menus, I would encourage you to open a feature-request issue. 🙂

**Reply by reinisg** [reply]
Thank you for this information!
